ivy/container/random.py

```diff
@@ -1090,6 +1108,24 @@ def shuffle(
         -------
         ret
             A container object, shuffled along the first dimension.
+
+        Examples
+        --------
+        With :class:`ivy.Container` inputs:
+
+        >>> x = ivy.Container(a=ivy.array([5,2,9]), \
+                              b=ivy.array([7,1,6]))
+        >>> x.shuffle()
+        {
+            a: ivy.array([9, 5, 2]),
+            b: ivy.array([6, 7, 1])
+        }
+
+        With :class:`ivy.Array` inputs:
```

Comment:
this example with `ivy.Array` only is not relevant to the Container instance method and should be removed 😅